db/migrate/20220114143800_migrate_has_representative_in_claims.rb

```diff
@@ -0,0 +1,15 @@
+class MigrateHasRepresentativeInClaims < ActiveRecord::Migration[6.1]
+  class Claims < ActiveRecord::Base
+    self.table_name = :claims
+  end
+
+  def up
+    Claim.all.each do |representative|
```

Comment:
Similar problem to another data migration here

1. Use of 'representative' as the variable name in the loop - but you are looping over claims when you look ?
2. You are effectively doing
```ruby

claim.update_columns has_representative: claim.has_representative.present?

```

Ill walk you through why this is wrong and why it is so important to check this value against what the calculated value was prior to this change.  If you were to run this, every single claim would have its has_representative field set to false.

1. You have added 'has_representative' to the claim schema - with no defaults, so the default is nil - every single claim at this point will have its has_representative set to nil
2. For each claim, your data migration sets the has_representative column to 'claim.has_representative.present?'.  As mentioned above, claim.has_representative will be nil at this point and 'nil.present?' will always return false.